### PR TITLE
Copy source tags

### PR DESCRIPTION
**Pull request by rickle-msft**

### CODE REVIEW

File: sdk/storage/azure-storage-blob/swagger/src/main/java/BlobStorageCustomization.java [hunk 1]
```diff
@@ -23,6 +23,11 @@ public void customize(LibraryCustomization customization, Logger logger) {
         PackageCustomization models = customization.getPackage("com.azure.storage.blob.models");
         models.getClass("PageList").addAnnotation("@JsonDeserialize(using = PageListDeserializer.class)");
 
+        models.getClass("BlobCopySourceTags").rename("BlobCopySourceTagsMode");
```
Comment (alzimmermsft) [inline sdk/storage/azure-storage-blob/swagger/src/main/java/BlobStorageCustomization.java:26]: Could this be done with a Swagger transform?

File: sdk/storage/azure-storage-blob/swagger/src/main/java/BlobStorageCustomization.java [hunk 2]
```diff
@@ -23,6 +23,11 @@ public void customize(LibraryCustomization customization, Logger logger) {
...
+        models.getClass("BlobErrorCode").getProperty("SNAPHOT_OPERATION_RATE_EXCEEDED")
```
Comment (alzimmermsft) [inline sdk/storage/azure-storage-blob/swagger/src/main/java/BlobStorageCustomization.java:27]: Do we want to keep this knowing that the method will throw based on testing it? Maybe comment it out for now until it is fix?

**Review by alzimmermsft — APPROVED** [review]

File: sdk/storage/azure-storage-blob/swagger/src/main/java/BlobStorageCustomization.java (hunk 1 above)
Comment (rickle-msft) [reply sdk/storage/azure-storage-blob/swagger/src/main/java/BlobStorageCustomization.java:26]: Is that preferable to a customization?

File: sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/models/BlobErrorCode.java [hunk 3]
```diff
@@ -375,6 +369,9 @@ public final class BlobErrorCode extends ExpandableStringEnum<BlobErrorCode> {
     public static final BlobErrorCode AUTHORIZATION_RESOURCE_TYPE_MISMATCH =
             fromString("AuthorizationResourceTypeMismatch");
 
+    /** Static value SnaphotOperationRateExceeded for BlobErrorCode. */
+    public static final BlobErrorCode SNAPHOT_OPERATION_RATE_EXCEEDED = fromString("SnaphotOperationRateExceeded");
+
```
Comment (kasobol-msft) [inline sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/models/BlobErrorCode.java:374]: why did we un-deprecated?

**Review by kasobol-msft — APPROVED** [review]

File: sdk/storage/azure-storage-blob/swagger/src/main/java/BlobStorageCustomization.java (hunk 2 above)
Comment (rickle-msft) [reply sdk/storage/azure-storage-blob/swagger/src/main/java/BlobStorageCustomization.java]: Removed

File: sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/models/BlobErrorCode.java (hunk 3 above)
Comment (rickle-msft) [reply sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/models/BlobErrorCode.java:374]: I think this was probably an oversight after generating. We're waiting on expanded customization abilities to handle deprecating this automatically.

**Pull request merged**